### DISCUSSION TITLE
I wonder disconnect wallet method?

### DISCUSSION THREAD

**Discussion by matthew2099** [WalletConnect/WalletConnectUnity#220] [Q&A]
I write my code

![image](https://github.com/WalletConnect/WalletConnectUnity/assets/109068040/400d3f5a-4592-4020-80ad-ef36ae984943)

But my wallet still have dapp.

How i do?

**Comment by skibitsky** [comment]
That's the correct way. There must be a bug in the wallet. Please report it to the wallet developer.

**Reply by matthew2099** [reply]
Ah. How i do resport it??
